paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/InputAddressScreen.kt

```diff
@@ -48,13 +44,6 @@ internal fun InputAddressScreen(
                 modifier = Modifier.padding(bottom = 8.dp)
             )
             formContent()
-            if (collectedAddress == null) {
```

Comment:
This screen also holds the unexpanded address form right? So I think need to have this button still. Did something change?

Reply:
iOS did not implement this, as we discussed offline. Will remove for now, we can add it back in later.